### DISCUSSION TITLE
library doesn't print verbose information

### DISCUSSION THREAD

**Discussion by dhirajsuvarna** [abetlen/llama-cpp-python#364] [General]
Hi, 
I am using llama-cpp-python to load a LLM model and want to see its running times during prediction. 

For this i am instantiating the model in the following manner - 
![image](https://github.com/abetlen/llama-cpp-python/assets/4989127/5ca742bc-4932-4eb7-b676-cc28c770ccef)

but i am not able to see any output regarding the timings. 
I am doing this on google colab. 
Am I missing something?

**Comment by Srini138** [comment]
@dhirajsuvarna Hi What model your using in model_path?

**Reply by dhirajsuvarna** [reply]
any LLM model which i download using Hugging Face Hub
